src/ui/components.tsx

```diff
@@ -176,10 +176,10 @@ export const CheckMkSelectNegatable = <T extends RequestSpecNegatableOptionKeys>
           label={label}
           autocompleter={autocompleter}
           onChange={onValueChange}
-          value={value.value}
+          value={value.value === '' ? undefined : value.value}
         />
       </InlineField>
-      <Checkbox label="Negate" value={value !== undefined ? value.negated : false} onChange={onNegateChange} />
+      <Checkbox label="Negate" value={value.negated} onChange={onNegateChange} />
```

Comment:
can you add a comment to the commit message mentioning this change? this seems to be unrelated to the current commit message?